### PR TITLE
Add Box.com and Dropbox.com as Blockable services

### PR DESCRIPTION
**Pull request by paragm**
This list is created based on respective services officially published domain list on their website

### CODE REVIEW

File: services/box.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+id: box
+name: Box
+rules:
+  - '||box.com^'
+  - '||box.net^'
+  - '||boxcloud.com^'
+  - '||boxcdn.net^'
+  - '||app.box.com^'
```
Comment (Sergey-Lyapin) [inline services/box.yml:8]: Subdomains are not needed. There's already box.com

File: services/dropbox.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  - '||files-uk.dropbox.com^'
+  - '||showbox-tr.dropbox.com^'
+  - '||showbox-eu.dropbox.com^'
+  - '||showbox-au.dropbox.com^'
+  - '||showbox-jp.dropbox.com^'
+  - '||showbox-uk.dropbox.com^'
+  - '||photos.dropbox.com^'
+  - '||dropbox.com^'
+  - '||bolt.dropbox.com^'
+icon_svg: <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 30 25"><path d="M7.70076 0.320312L0.478516 4.91332L7.70076 9.50633L14.9242 4.91332L22.1465 9.50633L29.3687 4.91332L22.1465 0.320312L14.9242 4.91332L7.70076 0.320312Z" /><path d="M7.70076 18.6925L0.478516 14.0994L7.70076 9.50633L14.9242 14.0994L7.70076 18.6925Z" /><path d="M14.9242 14.0994L22.1465 9.50633L29.3687 14.0994L22.1465 18.6925L14.9242 14.0994Z" /><path d="M14.9242 24.8164L7.70077 20.2234L14.9242 15.6304L22.1465 20.2234L14.9242 24.8164Z" /></svg>
```
Comment (Sergey-Lyapin) [inline services/dropbox.yml:79]: Please use this svg instead:
```
<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 -2.5 30 30"><path d="M7.7.32.48 4.92 7.7 9.5l7.22-4.6 7.23 4.6 7.22-4.6L22.15.32l-7.23 4.6L7.7.31Zm0 18.38L.48 14.1 7.7 9.5l7.22 4.6-7.22 4.6Z"/><path d="m14.92 14.1 7.23-4.6 7.22 4.6-7.22 4.6-7.23-4.6Zm0 10.72-7.22-4.6 7.22-4.59 7.23 4.6-7.23 4.59Z"/></svg>
```

File: services/dropbox.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  - '||dropboxinsiders.com^'
+  - '||dropboxlegal.com^'
+  - '||dropboxmail.com^'
+  - '||dropboxpartners.com^'
+  - '||dropboxstatic.com^'
+  - '||dropboxteam.com^'
+  - '||dropbox.tech^'
+  - '||dropbox.zendesk.com^'
+  - '||getdropbox.com^'
+  - '||learn.dropbox.com^'
```
Comment (Sergey-Lyapin) [inline services/dropbox.yml:24]: Subdomains are not needed, please remove them

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: services/dropbox.yml (hunk 3 above)
Comment (Alex-302) [reply services/dropbox.yml]: `dropbox.zendesk.com` should be with a subdomain

Comment (Alex-302) [reply services/dropbox.yml]: Maybe. Not a problem, if it will be blocked if whole Dropbox blocked.

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**